src/elements/DestructiveDialog/DestructiveDialog.less

```diff
@@ -0,0 +1,32 @@
+/**
+ * Destructive Dialog styles
+ */
+
+ @ui-destructive-dialog-font: @ui-font-sans-serif;
+ @ui-destructive-dialog-letter-spacing: 0.02em;
+
+.ui-destructive-dialog-container {
+	font-family:  @ui-destructive-dialog-font;
+	letter-spacing:  @ui-destructive-dialog-letter-spacing;
+	height: 186px;
+	width: 280px;
+
+	.destructive-dialog-title {
+		font-size: 20px;
+	}
+
+	.destructive-dialog-content {
+		color: @ui-color-gray-dark;
+		font-size: 16px;
+	}
+
+	.destructive-dialog-button-wrapper {
+		position: relative;
+		left: 100px;
+
+		button {
+			margin-right: 20px;
```

Comment:
Change this to a `margin-left` instead so that the rightmost button is flush to the right of the container. Also, let's use our `@ui-medium` space variable instead of explicitly giving it a pixel width.